style.css

```diff
@@ -233,6 +233,178 @@ div p {
   align-self: flex-start;
 }
 
+/* Popup section */
+@media (max-width: 767px) {
+  #imgD {
+    display: none;
+  }
+
+  #imgM {
+    justify-content: center;
+  }
+
+  .text_button {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .popup-buttons {
+    margin-top: 5%;
+  }
+
+  #pop {
+    justify-content: center;
+  }
+
+  .img_description {
+    justify-content: center;
+  }
+
+  .desktop-version {
+    display: none;
+  }
+}
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that the popup is responsive, it fits mobile and desktop sizes properly.So Kindly fix the responsiveness for the mobile screen size.
- Your Project Pop-up for mobile screen size Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/209284809-fa44baa4-3b07-41e6-b3eb-8e7a2c002bd8.PNG)

-  The Figma Pop-up for Mobile Screen size Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/209286084-601da19e-0179-4007-b0b7-dad0ad101abf.PNG)

Reply:
Hi, thank you for pointing it out. I was using the Dimensions of iPhone SE which is why there were no problems on my end. To resolve this, I have added the overflow property to my CSS file